be1-go/crypto/README.md

```diff
@@ -0,0 +1,14 @@
+# RSA Key pair for the PoPCHA system
+
+## Use
+
+The RSA key pair is used in the PoPCHA protocol for signing and
+verifying JWTs (Json Web Tokens). These JWTs define the format
```

Comment:
Maybe adding the adherence to RS256 and giving the reference URL : [](https://datatracker.ietf.org/doc/html/rfc7518#section-3.3) might be good ?